cardano-config/src/Cardano/TracingOrphanInstances/Common.hs

```diff
@@ -70,3 +76,196 @@ deriving newtype instance FromJSON EpochNo
 instance ToObject Void where
   toObject _verb x = case x of {}
 
+-- Common failures
+
+instance ToObject (PredicateFailure (DELEG c)) where
+  toObject _verb (StakeKeyAlreadyRegisteredDELEG alreadyRegistered) =
+    mkObject [ "kind" .= String "StakeKeyAlreadyRegisteredDELEG"
+             , "Credential" .= String (textShow alreadyRegistered)
+             , "error" .= String "Staking credential already registered"
```

Comment:
we should probably pick a consistent uppercase vs lowercase convention

Reply:
We should be using `camelCase` for JSON keys.